README.rst

```diff
@@ -31,58 +31,64 @@ Installation
 
 .. code-block:: console
 
-    (<custom virtualenv>)$ pip install invenio-cli
+    $ pip install invenio-cli
 
 Usage
 =====
 
-
 Local Development environment
 -----------------------------
 
 .. code-block:: console
 
     # Initialize environment and cd into <created folder>
-    (<custom virtualenv>)$ invenio-cli init --flavour=RDM
-    (<custom virtualenv>)$ cd <created folder>
+    $ invenio-cli init --flavour=RDM
+    $ cd <created folder>
 
-    # Install python dependencies, link/copy assets + statics, install js
+    # Install locally
+    # install python dependencies, link/copy assets + statics, install js
     # dependencies, build assets and final statics
-    (<custom virtualenv>)$ invenio-cli build --local --pre
+    $ invenio-cli install
 
     # Start and setup services (database, Elasticsearch, Redis, queue)
-    (<custom virtualenv>)$ invenio-cli services --local
+    $ invenio-cli services
 
     # Optional: add demo data
-    (<custom virtualenv>)$ invenio-cli demo --local
+    $ invenio-cli demo --local
 
     # Run the server
-    (<custom virtualenv>)$ invenio-cli runit
+    $ invenio-cli run
 
     # Update assets or statics
-    (<custom virtualenv>)$ invenio-cli update --local
-
-    # Get more help
-    (<custom virtualenv>)$ invenio-cli --help
+    $ invenio-cli update
 
 
 Containerized 'Production' environment
 --------------------------------------
 
-Just like the above, except with ``--containers``:
-
 .. code-block:: console
 
     # Initialize environment and cd into <created folder>
-    (<custom virtualenv>)$ invenio-cli init --flavour=RDM
-    (<custom virtualenv>)$ cd <created folder>
+    $ invenio-cli init --flavour=RDM
+    $ cd <created folder>
+
+    # Spin-up InvenioRDM
+    $ invenio-cli containerize
 
-    # Build Docker images
-    (<custom virtualenv>)$ invenio-cli build --containers --pre --verbose
+    # Optional: add demo data
+    $ invenio-cli demo --containers
 
-    # Setup services (database, Elasticsearch, Redis, queue)
-    (<custom virtualenv>)$ invenio-cli setup --containers --verbose
+    # After updating statics or code, if you do not need to re-install JS
+    # dependencies which can take time
+    $ invenio-cli containerize --no-install-js
```

Comment:
minor: For other parts we were using "skip" like "--skip-lock" in the *build* command (now *install*). So either we switch to `--no-lock` or `--skip-js` here? WDYT?

Reply:
Oh interesting, I didn't notice that parallel: the `--skip-lock` is because that's what pipenv uses (we don't want to deviate from that and confuse people). I don't know that there is an expectation for `--no-install-js` to have a matching `--no-lock`... I could be swayed if other people think so.

I have noticed something related to that while trying out the workflow though:

- `invenio-cli containerize` should default to installing the JS dependencies, because that is the spirit of the `containerize` command: rebuild everything (except for DB, indices...). But, we want the option to skip re-installing JS dependencies.

- `invenio-cli update` should default to NOT installing the JS dependencies, because most of the time local files are changed rather than new dependencies installed. But we want the option to force the installation of new JS dependencies.

So we do want an install and don't install options in both commands but their default would be opposite... What are your thoughts on that?

Reply:
I think it makes sense. In addition I do not see a problem with defaults being opposite (as long as is documented). My point being: a "default" is supposed to be that value that is used the most, in order to avoid typing  (we are lazy by nature)... So I think it all makes sense :)